docs/deploy.md

```diff
@@ -47,8 +47,10 @@ The following 15 flags are the most commonly used to control cert-exporter behav
     	Globs to match against secret data keys (Default "*").
   -secrets-label-selector value
     	Label selector to find secrets to publish as metrics.
-  -secrets-namespace string
+  -secrets-namespace string # (Deprecated) Use `-secrets-list-of-namespaces`.
     	Kubernetes namespace to list secrets.
+  -secrets-list-of-namespaces string
+        Kubernetes list of namespaces to search for secrets.
```

Comment:
I prefer the old name `-secrets-namespaces` and `-configmaps-namespaces`.

Reply:
These names are too similar and can be confused. But if you want, I can return the old names.